### PR TITLE
Update `grunt dev` to use `browserify-inc` rather than `watchify`, open playground page on first run

### PR DESCRIPTION
**Pull request by bcjordan**

### CODE REVIEW

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -318,17 +326,20 @@ module.exports = function (grunt) {
...
-      ' -o ' + allFilesDest[0]);
+  var browserifyExec = 'mkdir -p build/browserified && `npm bin`/browserifyinc' +
+      ' --cachefile ' + outputDir + 'browserifyinc-cache.json' +
+      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      ' -d ' + allFilesSrc.join(' ') +
+      (
+          APPS.length > 1 ?
+          ' -p [ factor-bundle -o ' + allFilesDest.join(' -o ') + ' ] -o ' + outputDir + 'common.js' :
+          ' -o ' + allFilesDest[0]
+      );
```
Comment (islemaster) [inline apps/Gruntfile.js:337]: +1 to improved formatting.

File: apps/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "test": "grunt test",
     "clean": "grunt clean",
     "build": "MOOC_DEV=1 grunt build",
-    "build:dist": "MOOC_DEV=0 grunt clean build"
+    "build:dist": "MOOC_DEV=0 grunt clean build",
+    "start": "MOOC_DEV=1 grunt dev"
```
Comment (islemaster) [inline apps/package.json:16]: Didn't I already do this?

File: apps/package.json [hunk 3]
```diff
@@ -59,9 +60,11 @@
     "grunt-lodash": "0.3.0",
     "grunt-newer": "1.1.1",
     "grunt-notify": "0.4.1",
+    "grunt-open": "^0.2.3",
     "grunt-sass": "1.0.0",
     "grunt-strip-code": "0.1.2",
     "http-proxy": "1.11.1",
+    "jit-grunt": "^0.9.1",
```
Comment (bcjordan) [inline apps/package.json]: Currently un-used in the gruntfile. It sped the grunt dev build time up .5-1.5s, but messed with the grunt timing output a bit. Want to do a bit more experimentation before pushing. I'll remove from gruntfile

Comment (islemaster) [inline apps/package.json]: What's this?  Does it get used?

File: apps/package.json (hunk 2 above)
Comment (bcjordan) [inline apps/package.json:16]: That must be the conflict

**Comment by islemaster** [comment]
Can we remove the watchify dependency in our package.json with this change?

**Comment by islemaster** [comment]
LGTM!

**Pull request merged**